schwifty/bank_registry/manual_fr.json

```diff
@@ -2398,5 +2374,21 @@
         "name": "SWAN",
         "short_name": "SWAN",
         "primary": true
+    },
+    {
+        "country_code": "FR",
+        "primary": true,
+        "bank_code": "AECF",
```

Comment:
Also (this is the point you already brought up probably) but why do we have letters for the bank codes when their banks have numbers?